tools/inventory-extract-windows/main.go

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2021 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/diskfs/go-diskfs/filesystem/fat32"
+	"github.com/diskfs/go-diskfs/partition"
+)
+
+var dirFlag = flag.String("dir", "", "save file to provided directory (will generate temp if empty)")
+var driveNumber = flag.Uint("drive", 0, "open drive with provided number")
+var showDrives = flag.Bool("list", false, "list drives")
+var helpFlag = flag.Bool("help", false, "help")
+
+func main() {
+	flag.Parse()
+	if *helpFlag {
+		fmt.Println("This program do export of INVENTORY partition from EVE drive")
+		fmt.Println("You can see list of available drives with -list flag")
+		fmt.Println("Then you must set Index of drive to -drive flag")
+		fmt.Println("With -dir flag you can provide directory to save files into. " +
+			"The program will create directory in temp if you will not provide this flag.")
+		return
+	}
+	if *driveNumber == 0 {
+		fmt.Println("Please provide -drive flag with Index of device from the list below:")
+		*showDrives = true
+	}
+	if *showDrives {
+		res, err := exec.Command("wmic", "diskdrive",
+			"where", "MediaType='Removable Media'",
+			"get", "index,caption").CombinedOutput()
+		if err != nil {
+			log.Println(err)
+		} else {
+			fmt.Println(string(res))
+		}
+		time.Sleep(time.Second * 5)
```

Comment:
what is this sleep for?

Reply:
Actually, in case of running it from non-administrative terminal, the program will prompt for Administrative privileges and re-open itself in new window. Without sleeps it will disappears immediately. I put little timers here and there to keep the windows on the screen for the little time.